unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -56,6 +59,11 @@ public class ExploreV2MenuComponentController : IExploreV2MenuComponentControlle
 
     public void Initialize()
     {
+        #if UNITY_EDITOR
```

Comment:
Why it is for Unity Editor only? What about builds?

Reply:
This change is in the scope of https://github.com/decentraland/unity-renderer/issues/4304 -- I left the TODO in there to analyze if it should be done also for the build, as 500 instances created looked a little too much IMHO but I'm not familiar with how are they getting used